Source/Scene/Scene.js

```diff
@@ -1274,6 +1274,7 @@ define([
         });
 
         var newMain =
+            //language="glsl"
```

Comment:
You can remove this line now. (Same with other the other areas that changed back)